modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -229,7 +232,7 @@ sub fetch_genome_report {
         @updated_annotations = grep { exists $allowed_species->{$_} } @updated_annotations;
     }
 
-    return ([@new_genomes, @updated_assemblies], \%renamed_genomes, \@updated_annotations);
+    return ([@new_genomes, @updated_assemblies, @updated_annotations], \%renamed_genomes, \@updated_annotations);
```

Comment:
We won't ever be running `PrepareMasterDatabaseForRelease` on this branch. There are soooo many other changes here that would break that pipeline long before this would be a problem! 😄  The main site will always be maintained by the production branches, as is documented in [Git Branches](https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EnsCom&title=Git+branches). These branches are expected to diverge.